app/controllers/test/oidc_test_controller.rb

```diff
@@ -0,0 +1,154 @@
+require './spec/support/oidc_auth_helper'
+module Test
+  class OidcTestController < ApplicationController
+    include OidcAuthHelper
+
+    BIOMETRIC_REQUIRED = 'biometric-comparison-required'
+
+    def initialize
+      @client_id = 'urn:gov:gsa:openidconnect:sp:test'
+      @update_redirect_uri = true
+      super
+    end
+
+    def index
+      # default to require
+      @start_url_selfie = "#{test_oidc_auth_request_url}?ial=biometric-comparison-required"
+      @start_url_ial2 = "#{test_oidc_auth_request_url}?ial=2"
+      @start_url_ial1 = "#{test_oidc_auth_request_url}?ial=1"
+      update_service_provider if @update_redirect_uri
+    end
+
+    def auth_request
+      ial = prepare_step_up_flow(ial: params[:ial])
+
+      idp_url = authorization_url(
+        ial: ial,
+        aal: params[:aal],
+      )
+
+      Rails.logger.info("Redirecting to #{idp_url}")
+
+      redirect_to(idp_url)
+    end
+
+    def auth_result
+      redirect_to('/')
+    end
+
+    def logout
+      redirect_to(logout_uri)
+    end
+
+    def authorization_url(ial:, aal: nil)
+      authorization_endpoint = openid_configuration[:authorization_endpoint]
+      params = ial2_params(
+        client_id: client_id,
+        acr_values: acr_values(ial: ial, aal: aal),
+        biometric_comparison_required: ial == BIOMETRIC_REQUIRED,
+        state: random_value,
+        nonce: random_value,
+      )
+      request_params = params.merge(
+        {
+          scope: scopes_for(ial),
+          redirect_uri: test_oidc_auth_result_url,
+        },
+      ).compact.to_query
```

Comment:
can remove optional curly braces

```suggestion
      request_params = params.merge(
        scope: scopes_for(ial),
        redirect_uri: test_oidc_auth_result_url,
      ).compact.to_query
```

Reply:
Changed.